h/static/scripts/directive/annotation.coffee

```diff
@@ -163,6 +163,12 @@ AnnotationController = [
         this.render()
         this.view()
 
+    this.group = ->
+      if model.group
+        group.getGroup(model.group)
+      else
+        group.focusedGroup()
```

Comment:
This feels wrong to me. We need a clearer upgrade path here. Right now, old annotations will appear to switch groups to match whatever the focused group is.